test/libsolidity/syntaxTests/viewPureChecker/call_internal_functions_success.sol

```diff
@@ -5,4 +5,3 @@ contract C {
     function i() payable public { i(); h(); g(); f(); }
 }
 // ----
-// Warning 6321: (89-93): Unnamed return variable can remain unassigned. Add an explicit return with value to all non-reverting code paths or name the variable.
```

Comment:
These errors are gone because of the unconditional recursion?

Reply:
correct